src/index.html

```diff
@@ -7,6 +7,52 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
```

Comment:
So if you create your calendar with additional header it would be nice to change your selectors a little
```suggestion
    <div class="calendar__container">
```